rules/composer/src/ValueObject/MovePackage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Composer\ValueObject;
+
+use Rector\Composer\Modifier\ComposerModifier;
+use Rector\Composer\Contract\ComposerModifier\ComposerModifierInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * Moves package to selected section, if package is not in composer data, nothing happen, also if package is already in selected section
+ * @see \Rector\Composer\Tests\ValueObject\MovePackageTest
+ */
+final class MovePackage implements ComposerModifierInterface
```

Comment:
Move package where? 

I see from code bellow the require/require-dev is the target. It would be better to have 2 objects, that could explain this in a class name. E.g. MovePackageToRequire, MovePackageToRequireDev

Reply:
Ok I will split it to two classes

Reply:
AddPackage has also this $section flag in __construct. Should I split it too?

Reply:
Yes, makes sense